src/DDTrace/Util/Versions.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace DDTrace\Util;
+
+/**
+ * Utility functions to handle version numbers and matching.
+ */
+final class Versions
+{
+    /**
+     * @param string $version
+     * @return bool
+     */
+    public static function phpVersionMatches($version)
+    {
+        return self::versionMatches($version, phpversion());
+    }
+
+    /**
+     * @param string $expected
+     * @param string $specimen
+     * @return bool
+     */
+    public static function versionMatches($expected, $specimen)
+    {
+        $expectedFragments = self::asIntArray($expected);
+        $specimenFragments = self::asIntArray($specimen);
+
+        if (empty($expectedFragments) || empty($specimenFragments)) {
+            return false;
+        }
+
+        for ($i = 0; $i < count($expectedFragments); $i++) {
```

Comment:
Slight optimization. :)
```suggestion
        $count = count($expectedFragments);
        for ($i = 0; $i < $count; $i++) {
```